index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
+        <div class="navbar_menu">
+          <div class="navbar_menu_button">
+            <img src="assets/menu_icon.svg" alt="menu_icon">
+          </div>
+        </div>
+        
+      </div>
+      
+    </div>
+    <div class="main_section">
+      <h1 class="main_section_h1">Hey there. I'm Koami.</h1>
+      <h2 class="main_section_h2">I'm a software developer</h2>
```

Comment:
- [ ] Kindly use the exact colors for the title as used in the Figma design template. It's a good practice to always stick to design as much as possible 👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/185479417-52beb3e7-3834-46bf-b504-39d9594bdcc5.png)  |![image](https://user-images.githubusercontent.com/63473584/185479560-d73892f4-a5e6-4d5b-a8a4-33e09ce91454.png)  |

Reply:
thanks, @vickymarz, please I updated it.